### ISSUE TITLE
Normalized output for `B.D’Orbigny` should not add a space after `D'`

### ISSUE THREAD

**Issue by dimus** [gnames/gnparser#252]
```json
"authorship": {
    "verbatim": "B.D’Orbigny",
    "normalized": "B. D' Orbigny",
    "authors": [
      "B. D' Orbigny"
    ],
    "originalAuth": {
      "authors": [
        "B. D' Orbigny"
      ]
    }
  },
```

**Comment by dimus** [comment]
As a related issue, I will fix `Acteonina d&#039;Orbigny 1849`

**Comment by dimus** [comment]
Cannot reproduce, html entities are already taken care of
